addons/point_of_sale/static/tests/pos/tours/pricelist_tour.js

```diff
@@ -23,10 +23,10 @@ registry.category("web_tour.tours").add("pos_pricelist", {
             OfflineUtil.setOfflineMode(),
             ProductScreen.clickPriceList("Fixed", true, "Public Pricelist"),
             ProductScreen.clickPartnerButton(),
-            ProductScreen.clickCustomer("Deco Addict"),
+            ProductScreen.clickCustomer("Deco Addict", false),
```

Comment:
`false` not needed as it should be the default value

```suggestion
            ProductScreen.clickCustomer("Deco Addict"),
```